src/server/game/Guilds/Guild.cpp

```diff
@@ -1440,6 +1440,14 @@ void Guild::HandleInviteMember(WorldSession* session, std::string const& name)
     if (pInvitee->GetSocial()->HasIgnore(player->GetGUID()))
         return;
 
+    uint32 memberLimit = sConfigMgr->GetOption<uint32>("Guild.MemberLimit", 0);
+    if (memberLimit > 0 && player->GetGuild()->GetMemberCount() >= memberLimit)
+    {
+        player->SendSystemMessage("Guild has the maximum amount of members. Cannot send another invite.");
```

Comment:
Maybe we could send the maximum amount within the message?

Reply:
I thought about it. Tested it and it didnt work. But maybe SendSystemMessage dont support fmt style?

Reply:
Nopers, you need to do as before, with %u

Reply:
Its PSendSystemMessage to send variables

Reply:
And its not supported by the player helper I added, so need to build it with the chathandler

Reply:
Making my life difficult D:

I will fix this before I merge

Reply:
Maybe we should expose it to player helper 🤷

Reply:
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/24550914/7850885c-5ef4-4d88-9fa3-b9e184bd61c7)